Implement stakers UI support (#21)
dappnode/DAppNodePackage-Besu: getting-started.md

## Besu (Execution Client)

> Welcome to your Post Besu Execution Layer Client

Reviewer: ```suggestion
Welcome to the Merged Besu Ethereum Mainnet Execution Layer Client
```
Not fixed and we need to say mainnet ethereum since we have on all packages so far, or at least I have when i was assigned a couple months ago.

There are now two RPC APIs in Execution Clients:

1. Querying API `http://besu.dappnode:8545`. Use this endpoint to query transactions on your node and connect to it with your web3 wallet.

> 2. Engine API `htt Mergep://besu.dappnode:8551`. Use this endpoint to connect your Beacon Chain (Consensus Layer) client.

Reviewer: ```suggestion
2. Engine API `http://besu.dappnode:8551`. Use this endpoint to connect your Beacon Chain (Consensus Layer) client.
```
I dont get it this looks like it was fine and now is nonsense that's not a link

> ## Goerli Besu (Execution Client)

Reviewer 2: Is this correct?

Author: fixed!

Reviewer: Mainnet not goerli

> Welcome to your Post Besu Execution Layer Client

Reviewer 2: Do you mean post-merge here?

Author: fixed!
